frontend/conf/application.conf

```diff
@@ -13,7 +13,7 @@ event.discountMultiplier=0.8
 
 content.api.key=""
 
-cors.allow.origin="https://profile.theguardian.com,http://www.theguardian.com"
+cors.allow.origin="https://profile.theguardian.com,http://www.theguardian.com,https://composer.gutools.co.uk,https://composer.code.gutools.co.uk,https://composer.qa.gutools.co.uk,https://composer.release.gutools.co.uk,https://composer.local.dev-gutools.co.uk"
```

Comment:
These should be `https://composer.code.dev-gutools.co.uk` (`dev-`) for all the pre-PROD envs.